app/lib/rocketchat/rocketchat.js

```diff
@@ -1066,21 +1016,7 @@ const RocketChat = {
 		return result?.records;
 	},
 	methodCallWrapper(method, ...params) {
-		const { API_Use_REST_For_DDP_Calls } = reduxStore.getState().settings;
-		const { user } = reduxStore.getState().login;
-		if (API_Use_REST_For_DDP_Calls) {
-			const url = isEmpty(user) ? 'method.callAnon' : 'method.call';
-			return this.post(`${url}/${method}`, {
-				message: EJSON.stringify({ method, params })
-			});
-		}
-		const parsedParams = params.map(param => {
-			if (param instanceof Date) {
-				return { $date: new Date(param).getTime() };
-			}
-			return param;
-		});
-		return this.methodCall(method, ...parsedParams);
+		return sdk.methodCallWrapper(method, ...params);
```

Comment:
I didn't want to create a separate file for this, because we're going to remove it after the migration anyway over the sdk version.
Same for `post` and `methodCall`